pwlwp/cache_main.py

```diff
@@ -20,7 +20,7 @@ def doit(index, sentence):
 	return sentence_transformer.encode(sentence)
 
 if __name__ == "__main__":
-	sentence_transformer_name: str = "all-distilroberta-v1"
+	sentence_transformer_name: str = "all-MiniLM-L6-v2"
 	input_file_name: str = "../corpora/causalBeliefSentences.tsv"
```

Comment:
I think these are still the small files.  We need to make sure this works with the large ones.

Reply:
This includes running the sentence transformer on a million sentences.  That probably needs to be done in advance.